ably/proto_message_decoding_test.go

```diff
@@ -0,0 +1,149 @@
+package ably
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/ably/ably-go/ably/internal/ablyutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type fixture struct {
+	Data          string `json:"data"`
+	Encoding      string `json:"encoding"`
+	ExpectedType  string `json:"expectedType"`
+	ExpectedValue any    `json:"expectedValue"`
+	ExpectedHex   []byte `json:"expectedHex"`
```

Comment:
I thought it was called `expectedHexValue`?

Reply:
well spotted.
Fixed